app/services/candidate_interface/decoupled_references/send_reference_reminder.rb

```diff
@@ -21,20 +21,24 @@ def call
 
           send_slack_message(application_form, message)
 
-          flash[:success] = "Reminder sent to #{reference_name}"
+          flash[:success] = "Reminder sent to #{referee_name}"
         end
       end
 
     private
 
       def message
-        "Candidate #{application_form.first_name} has sent a reminder to #{reference_name}"
+        "Candidate #{application_form.first_name} has sent a reminder to #{referee_first_name}"
```

Comment:
We can use `reference.first_name` instead, it's inputted separately.

Reply:
We just capture the whole name in a single field, unfortunately! https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/views/candidate_interface/decoupled_references/name/_shared_form.html.erb

Reply:
![](https://media.giphy.com/media/AKaEfzaLlr0yI/giphy-downsized.gif)